### DISCUSSION TITLE
界面高速刷新问题

### DISCUSSION THREAD

**Discussion by hinadesu39** [BYJRK/DotNet-Discussions#298] [WPF 开发]
大佬们，我现在有个需求，需要在主界面上以25帧的频率循环切换一组图片来显示，我用了异步，多线程，Timer，切换的时候都不卡，但是一旦拖动主窗体或者把鼠标移入Image控件中都会发生卡顿，打印cs代码中切换图片的相关逻辑中切换发生的那一瞬间的时间戳发现确实时40毫秒一次，并没有卡顿，猜测问题出在界面的某些逻辑上，请问有什么好的解决方法吗

**Comment by hinadesu39** [comment]
不用vs启动程序，直接exe启动居然不卡

**Reply by BYJRK** [reply]
是因为你在 VS 中开启调试器导致的卡顿？

**Reply by hinadesu39** [reply]
就正常debug模式启动，其他啥都没干
